rust/src/domains/polars/frame/mod.rs

```diff
@@ -210,6 +210,10 @@ impl<F: Frame> FrameDomain<F> {
         self.margins.insert(grouping_keys, margin);
         Ok(self)
     }
+
+    pub fn schema(&self) -> Schema {
```

Comment:
I think this is only used for tests? Could imagine it being defined for a narrower scope.

Reply:
It will also be used in the context PR.